HexRaysPyTools/callbacks/actions.py

```diff
@@ -65,7 +65,9 @@ def activate(self, ctx):
         # type: (idaapi.action_activation_ctx_t) -> None
         raise NotImplementedError
 
-    def check(self, hx_view: idaapi.vdui_t):
+    def check(self, hx_view):
+        """hx_view: idaapi.vdui_t
+        """
```

Comment:
One more thing - there's python2 type [hinting style](https://www.python.org/dev/peps/pep-0484/) which for example PyCharm understands. So it should be:

```python
    def check(self, hx_view):
       # type: (hx_view: idaapi.vdui_t) -> bool
```